src/data/blog/Hiring-web-dev-092019.md

```diff
@@ -26,44 +26,59 @@ author_facebook: null
 author_twitter: null
 ---
 
-##Want to (web) develop your career with us?  
+##Do good with us  
 
-We're a software development company working on web user interfaces and the APIs that power them. We've got a national reputation for our work in tech-for-good, an international reputation in travel and tourism, and right now we have more and more work coming our way.  
+Neontribe is a software development company working on web user interfaces and the APIs that power them. We've got a national reputation for our work in tech-for-good, and right now we have more and more work coming our way.
 
-Right now, we’re recruiting to fill a specific role in our Norwich team. However, we’re also always on the look out for people who would make our teams stronger, so if this doesn’t suit you please see the information at the end.
+We’re recruiting to fill a specific role in our team. If you want to spend time in an office, we've space in Exeter and Norwich, but we're very happy with remote working. We’re also always on the lookout for people who would make our teams stronger, so if this doesn’t suit you please see the information at the end.
 
-##Working mainly in HTML/CSS and Javascript  
+We’re looking for someone who:
 
-You'll have a flair for implementing user interfaces using web technologies. You'll fully understand that what you are designing are interactions, on screens that vary in size, not fixed images on a fixed canvas and enjoy producing interfaces that respond to user needs. Your tools will be HTML, CSS and Javascript and you’ll be passionate about how they work together. If you’re familiar with React, that would be an advantage. You’ll know how to work in a team with several developers and you may already have led some projects or project elements and will be keen to do more team leading.
+- works mainly in HTML/CSS and Javascript
+- has a flair for implementing user interfaces using web technologies. 
+- understands they are developing interactions, on screens that vary in size, that must be accessible to all their users. 
+- is passionate about how  HTML/CSS and Javascript work together, and familiarity with React would be an advantage0
```

Comment:
You have a rogue 0 at the end of the sentence.

Reply:
Yes, damnit! Thank you!

...I've got to remember how to edit my commit now...